docs/inline-svgs.md

```diff
@@ -96,7 +98,7 @@ The result of inserting an SVG with either method is the compressed source wrapp
 So both this Scala:
 
 ```scala
-@fragments.inlineSvg("profile-36", "icon", List("rounded-icon", "control__icon-wrapper"))
+@fragments.inlineSvg("profile-36", "icon", List("rounded-icon", "control__icon-wrapper"), Some("Profile icon"))
```

Comment:
shoudl this be a `some`? (i don't know, it looks weird to me, but I really don't know)